tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/op/core/Init.java

```diff
@@ -1,110 +0,0 @@
-package org.tensorflow.op.core;
-
-import org.tensorflow.ExecutionEnvironment;
-import org.tensorflow.Graph;
-import org.tensorflow.Operation;
-import org.tensorflow.OperationBuilder;
-import org.tensorflow.op.Op;
-import org.tensorflow.op.RawOp;
-import org.tensorflow.op.Scope;
-import org.tensorflow.op.annotation.Endpoint;
-import org.tensorflow.op.annotation.Operator;
-
-@Operator
-public final class Init extends RawOp {
```

Comment:
After this class has been removed how do I find the initializers of a Graph when I load it from a GraphDef? Before I could find the name of this op and save it out with the GraphDef, allowing me to run the initialization when all I have is a GraphDef. Without it I'm going to have to mirror the logic into user code and then have my own function which makes a `NoOp` with the required control dependencies.